Brokerages/Backtesting/BacktestingBrokerage.cs

```diff
@@ -250,14 +243,14 @@ public void Scan()
                     catch (Exception err)
                     {
                         Log.Error("BacktestingBrokerage.Scan(): " + err.Message);
-                        _algorithm.Error(string.Format("Order Error: id: {0}, Transaction model failed to fill for order type: {1} with error: {2}", order.Id, order.Type, err.Message));
+                        _algorithm.Error(string.Format("BackTestingBrokerage.Scan(): Order Error: id: {0}, Transaction model failed to fill for order type: {1} with error: {2}", order.Id, order.Type, err.Message));
```

Comment:
Please remove the method names from these error messages. These messages are user facing.